controller/src/main/java/org/apache/rocketmq/controller/ControllerManager.java

```diff
@@ -193,16 +119,18 @@ public void registerProcessor() {
     }
 
     public void start() {
-        this.heartbeatManager.start();
         this.controller.startup();
-        this.remotingClient.start();
+        this.brokerControllerManager.start();
     }
 
     public void shutdown() {
-        this.heartbeatManager.shutdown();
         this.controllerRequestExecutor.shutdown();
         this.controller.shutdown();
-        this.remotingClient.shutdown();
+        this.brokerControllerManager.stop();
```

Comment:
seems better to stop scanNotActiveBroker  scheduled task before controller stopped.